docker-compose.yml

```diff
@@ -72,6 +72,19 @@ services:
     depends_on:
       - vault
 
+  product_incentory_db:
+    image: postgres:13-alpine
+    container_name: ecomm-product-db
+    user: postgres
+    volumes:
+      - ./dispute_management_service/.postgres_data:/var/lib/postgresql/data
```

Comment:
- inventory*
- you may want to add a dependency to the "product_inventory" service
- the volume points to where the dispute management's database should be